MANIFEST

```diff
@@ -174,13 +270,95 @@ Makefile.PL
 MANIFEST			This list of files
 MANIFEST.SKIP
 README.md
-t/00_cds.t
-t/01_cmath.t
-t/02_mikoslides.t
-t/03_modules1.t
-t/03_modules2.t
-t/04_omtext.t
-t/05_statements.t
-t/06_smglom.t
-t/07_inlinedef.t
+
+% File in t/, these should be updated from time to time
```

Comment:
The manifest needs updating. Travis complains, but doesn't fail. @dginev do you know of a way to make travis fail when the MANIFEST isn't up-to-date?

Reply:
I don't know what the manifest does or how it is updated at all